apps/daffio/src/app/core/nav/sidebar-body/component.ts

```diff
@@ -0,0 +1,43 @@
+import { AsyncPipe } from '@angular/common';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  OnInit,
+} from '@angular/core';
+import { RouterLink } from '@angular/router';
+import {
+  map,
+  Observable,
+} from 'rxjs';
+
+import { DaffListModule } from '@daffodil/design/list';
+import { DaffRouterDataService } from '@daffodil/router';
+
+import { DaffioRouteWithNavLinks } from '../link/route.type';
+import { DaffioNavLink } from '../link/type';
+
+@Component({
+  selector: 'daffio-nav-links-sidebar-body',
```

Comment:
i think `daffio-docs-sidebar-body` is more indicative of what this component is rendering

Reply:
I disagree. This is rendered on the homepage as well so its not specific to docs. This is the sidebar that renders the list of nav links; its name describes its content, not its context.

Reply:
What happens if I end up wanting to add more content other than nav links to the docs sidebar?

Reply:
I see your point. You'd have to change the name of it. I still don't think `daffio-docs-sidebar-body` is correct though, this sidebar gets rendered on the homepage too.

I think of it as rendering whatever gets hidden from the header when the viewport shrinks sufficiently so maybe `daffio-header-sidebar-body` or something?

Reply:
I mean as in, why isn't the marketing and docs sidebar body in separate components? If I end up wanting to add information to the docs sidebar body that isn't shown in the marketing sidebar body, how will I go about doing that?

Reply:
you would need to override the registration (in the routes) that correspondes to the activator in the header (`DAFF_NAV_LINKS_SIDEBAR_REGISTRATION` at the time of writing). So I guess it really correspondes to the *source* of the sidebar activation. I will rework the naming scheme here.

Reply:
I updated the naming a bit but ultimately I think this is correct. The "nav sidebar" is named that because it originates from the nav sidebar activator. If the docs pages needed some other content for its nav sidebar, then you could create a "docs nav sidebar" and override the nav sidebar registration for docs routes.